spec/spec_helper.rb

```diff
@@ -1,3 +1,6 @@
+RSpec.configure do |c|
```

Comment:
is that required? It is not present in our modulesync

Reply:
This is from the spec_helper.rb.erb template provided by PDK templates (as we can't really use both the PDK template and the modulesync templates).

Reply:
Without this we get deprecation warnings. We probably need to do the same in our modulesync.

Reply:
https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/spec/spec_helper.rb.erb#L47 modulesync can already manage this for us, and it is also in the sync.yml